src/babylon.assetContainer.ts

```diff
@@ -1,29 +1,109 @@
 module BABYLON {
+    /**
+     * Set of assets to keep when moving a scene into an asset container.
+     */
+    export class KeepAssets {

```

Comment:
Not sure to understand why not lights and other asset types?

Reply:
I missed this. I will add the others and add the missing explicit public to there definitions,